examples/mxnet/train/byteps_dist_gpu_v1.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: "kubeflow.org/v1"
+kind: "MXJob"
+metadata:
+  name: "byteps-mxnet-job"
+spec:
+  jobMode: MXTrain
+  cleanPodPolicy: Running
```

Comment:
There's a little bit api change to these policies. 
Can you change to

```
runPolicy:
  cleanPodPolicy: Running
```

Can you delete `successPolicy ` as well? MxNet doesn't support it and I do think this is added by mistake in the past